Zend/Optimizer/zend_func_infos.h

```diff
@@ -134,7 +134,6 @@ static const func_info_t func_infos[] = {
 #if defined(HAVE_GD_BMP)
     F1("imagecreatefrombmp", MAY_BE_OBJECT|MAY_BE_FALSE),
 #endif
-    F0("imagecolorset", MAY_BE_FALSE|MAY_BE_NULL),
```

Comment:
It is generate by the ``build/gen_stub.php`` file with the ``--generate-optimizer-info`` option

Reply:
This line is removed because the entry doesn't have any additional info anymore for the optimizer (since the native type can now express the type exactly)

Reply:
Ok. I checked other methods that don't have additional info but are still in the list, but I might be missing something.

Reply:
Yeah, the other important part is the `F0`/`F1`/`FN` in the beginning of the line expressing the refcount number. As scalar types always have a refcount of 0, both the type and refcount info is now superfluous.